source/agora/consensus/EnrollmentManager.d

```diff
@@ -55,7 +55,7 @@ import agora.consensus.data.PreImageInfo;
 import agora.consensus.EnrollmentPool;
 import agora.consensus.PreImage;
 import agora.consensus.validation;
-import agora.consensus.state.ValidatorSet;
+public import agora.consensus.state.ValidatorSet;
```

Comment:
Deliberate change?

Reply:
Yep, `ExpiringValidator` is defined in ValidatorSet. That should be visible to modules using `EnrollmentManager.getExpiringValidators`